src/applications/find-forms/widgets/createInvalidPdfAlert.js

```diff
@@ -39,26 +49,41 @@ export async function onDownloadLinkClick(event) {
   // Default to true in case we encounter an error
   // determining validity through the API.
   let formPdfIsValid = true;
+  let formPdfUrlIsValid = true;
   let form = null;
 
   try {
     const forms = await fetchFormsApi(formNumber);
     form = forms.results.find(f => f.id === formNumber);
     formPdfIsValid = form?.attributes.validPdf;
+    if (formPdfIsValid) {
+      // URLS can be entered invalid, 400 is returned, this checks to make sure href is valid
+      const response = await fetch(downloadUrl);
+      if (!response.ok) formPdfUrlIsValid = false;
+    }
   } catch (err) {
-    // Todo
+    sentryLogger(
+      form,
+      formNumber,
+      downloadUrl,
+      'Find Forms - Form Detail - onDownloadLinkClick function error',
+    );
```

Comment:
Left a comment about this - the IE error I saw talked about the other day is from a call to `.remove`